fix: Indicator dot colors in dark mode

surajshetty3416 opened the pull request:
**Before:**
<img width="990" alt="Screenshot 2023-01-20 at 2 17 22 PM" src="https://user-images.githubusercontent.com/13928957/213654029-240656a3-3793-4f21-afb0-865e577e25eb.png">


**After:**
<img width="1002" alt="Screenshot 2023-01-20 at 7 22 01 PM" src="https://user-images.githubusercontent.com/13928957/213715036-534f03a0-fc37-4dad-acfb-5f065410f691.png">


triggered by: https://github.com/frappe/frappe/pull/19634

surajshetty3416 committed:
fix: Indicator dot colors in dark mode

ankush commented:
Check WCAG scores. Some of them aren't easy to read - like gray (light version) :grimacing: should at least get AA

ref https://webaim.org/resources/contrastchecker/ or https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf?hl=en

RJPvT commented:
After approval please backport v13 :)

barredterra commented:
To add to Ankush's comment:

- _cyan_ and _pink_ on black background look like white to me. They could use some more saturation.
- _gray_ and _dark gray_ on black background could both be much lighter

shariquerik commented on frappe/public/scss/common/indicator.scss:
```diff
@@ -67,111 +51,30 @@
-}
+$indicator-colors: green, cyan, blue, orange, yellow, gray, grey, red, pink, darkgrey, purple, light-blue;
+@each $color in $indicator-colors {
+	.indicator.#{"" + $color} {
```
why `#{"" + $color}`?🤨
at some place you kept `#{$color}`

surajshetty3416 replied:
@surajshetty3416 SCSS compiler throws warning when you try to or use a variable as selector name.

<img width="791" alt="Screenshot 2023-01-20 at 7 25 26 PM" src="https://user-images.githubusercontent.com/13928957/213714305-9eeb9bcb-98a5-4bf4-8857-e4b5f7711c6d.png">

surajshetty3416 committed:
fix: Indicator contrast to make it more readable

surajshetty3416 committed:
fix: Add cyan and pink indicator color for dark theme

- fix gray indicator colors on dark... make them lighter to make it more consistent and visible

surajshetty3416 commented:
> like gray (light version) 😬 should at least get AA

resolved with https://github.com/frappe/frappe/pull/19681/commits/e2b3dd60c28ba899a717fdc501d995befb47a4b8

surajshetty3416 commented:
> To add to Ankush's comment:
> 
> * _cyan_ and _pink_ on black background look like white to me. They could use some more saturation.
> * _gray_ and _dark gray_ on black background could both be much lighter

Made changes in https://github.com/frappe/frappe/pull/19681/commits/ee3a6fab7d61fd2d35d77c2a2be82baeaccef263 to fix it. You can check the updated description.
